components/scream/cime_config/namelist_defaults_scream.xml

```diff
@@ -162,7 +163,8 @@ tweaking their $case/namelist_scream.xml file.
     <!-- Simple Prescribed Aerosols (SPA) -->
     <spa inherit="physics_proc_base">
       <SPA__Remap__File>none</SPA__Remap__File>
-      <SPA__Data__File>${DIN_LOC_ROOT}/atm/scream/init/spa_file_unified_and_complete_ne4_scream.nc</SPA__Data__File>
+      <SPA__Data__File hgrid="ne4np4">${DIN_LOC_ROOT}/atm/scream/init/spa_file_unified_and_complete_ne4_20220428.nc</SPA__Data__File>
+      <SPA__Data__File hgrid="ne30np4">${DIN_LOC_ROOT}/atm/scream/init/spa_file_unified_and_complete_ne30_20220428.nc</SPA__Data__File>
     </spa>
```

Comment:
Don't we have some ne2 tests which need a SPA regrid file? Do we need to include the capability for that in this PR? I guess not because this PR doesn't break tests?